clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -182,9 +226,30 @@ const TcfOverlay: FunctionComponent<OverlayProps> = ({
     [draftIds]
   );
 
+  const privacyNotices = useMemo(
+    () => experience.privacy_notices ?? [],
+    [experience.privacy_notices]
+  );
+
+  const isAllNoticeOnly = privacyNotices.every(
+    (n) => n.consent_mechanism === ConsentMechanism.NOTICE_ONLY
+  );
```

Comment:
we don't need this right now—all TCF payloads will return an empty `privacy_notices` array, so we can just pass `[]` to `notices`. also, TCF doesn't have a concept of notice-only _yet_ (see https://github.com/ethyca/fides/issues/4141), so for now, let's just pass in false for `acknowledgeMode`